lib/workers/branch/index.ts

```diff
@@ -320,6 +326,60 @@ export async function processBranch(
     } else {
       logger.debug('No updated lock files in branch');
     }
+
+    if (global.trustLevel === 'high') {
+      logger.debug('Executing post-change hooks');
```

Comment:
```suggestion
      logger.debug('Checking for post-upgrade hooks');
```

Reply:
I've updated the logging statement with your suggestion